src/lib/logMsg/logMsg.cpp

```diff
@@ -343,6 +343,9 @@ int             inSigHandler      = 0;
 char*           progName;                   /* needed for messages (and by lmLib) */
 char            progNameV[512];             /* where to store progName            */
 __thread char   transactionId[64] = "N/A";
+__thread char   srv[50 + 1]       = "N/A";     // FIXME: Maybe we should use limits.h ?
+__thread char   subsrv[51*10 +1]  = "N/A";     // FIXME: Maybe we should use limits.h ?
+__thread char   from[4*4 + 1]     = "N/A";     // Based in XXX.XXX.XXX.XXX
 
```

Comment:
The names of the variables came after the names that Operaions have defined for the fields of the log.

```
srv= ... | subsrv= ... | from= ...
```

I think that having the same name will help to identify the purpose of each variable.

I'll move limits to limit.h